integration/spark2/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -248,7 +248,7 @@ object CarbonSession {
 
         session = new CarbonSession(sparkContext, None, !enableInMemCatlog)
         val carbonProperties = CarbonProperties.getInstance()
-        if (storePath != null) {
+        if (storePath != null && !storePath.trim.isEmpty) {
```

Comment:
why not use StringUtils or other String tools

Reply:
such as StringUtils.isNotEmpty

Reply:
StringUtils.isNotEmpty shouldn't support " "

Reply:
Use StringUtils.isNotBlank.